com.eKitabu.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This filename does not match the app ID. It should be `com.ekitabu.eKitabu.appdata.xml`.